src/main/java/edu/kis/powp/jobs2d/command/gui/A4Canvas.java

```diff
@@ -0,0 +1,40 @@
+package edu.kis.powp.jobs2d.command.gui;
+
+import javax.swing.*;
+import java.awt.Dimension;
+import java.awt.Point;
+
+
+public abstract class A4Canvas implements Canvas {
```

Comment:
Now it cannot be used at all :-/. Revert, or provide a "proof of life" (example usage)